<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HotelDialog</class>
 <widget class="QDialog" name="HotelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отель</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="countryLabel">
       <property name="text">
        <string>Страна:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="countryCombo"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="starsLabel">
       <property name="text">
        <string>Звезды:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="starsSpin">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Адрес:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="addressEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="roomsLabel">
     <property name="text">
      <string>Номера:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="roomsTable">
     <column>
      <property name="text">
       <string>Название</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Тип</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Цена за ночь</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Вместимость</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addRoomButton">
       <property name="text">
        <string>Добавить номер</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editRoomButton">
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteRoomButton">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

